### PR TITLE
Reorganize project files, add Bower configuration, add npm build scripts

### PR DESCRIPTION
**Pull request by jgarber623**
Hello, Scott and crew!

This PR reorganizes the project in a way that may make future development and testing easier. Here's a rundown of what I've done:
- Add an [EditorConfig](http://editorconfig.org) configuration file to enforce code style. I based the settings on what I noticed in the existing files.
- Add a CONTRIBUTING.md file. GitHub will show a link to this file when anyone opens a new PR. Wording for this file is based on some of my own projects and its contents is certainly up for debate.
- Update the README.md with mostly [bikeshed](https://en.wikipedia.org/wiki/Law_of_triviality)-y changes. I _did_ add a link to the CONTRIBUTING.md file and added a table with file sizes (gathered from the `stats.js` script). The file sizes table could be useful when developing and releasing future versions.
- Add [bower.json](https://github.com/bower/spec/blob/master/json.md) configuration file. See the section below for some of my reasoning for adding this.
- Moves source file to `src/loadJS.js` and adds a `dist` folder with `loadJS.js` and `loadJS.min.js` files.
- Adds a build script that can be run from the root of the project with `npm run build`. This script creates the files in the `dist` folder and adds the preamble (`/*! loadJS … */`). It also runs the stats script which displays the various sizes (in bytes) of the generated files.
- Adds a stats script that can be run from the root of the project with `npm run stats`.
## Why add Bower config?

[Bower](http://bower.io/)'s similar-but-different from npm as far as package managers go. A lot of FEDs use it so it'd be helpful I think to support it. I'm most interested in adding loadJS to Bower's registry so that I can then use [Rails Assets](https://rails-assets.org) to include it in my Rails projects via the [Asset Pipeline](http://guides.rubyonrails.org/asset_pipeline.html).

If you end up merging this PR and adding `bower.json` to the repo, you'll want to run the following command (after installing Bower, of course: `npm install -g bower`) to add loadJS to Bower's registry:

``` sh
bower register loadjs git://github.com/filamentgroup/loadJS.git
```

Publishing updated packages to Bower will happen automatically when new releases are pushed to GitHub (via [semver](http://semver.org)'ed tags).
## In conclusion…

If there are parts of this that you like, I'd be happy to do a separate PR with just those parts. This PR certainly makes the project's organization consistent with how I tend to work and I can totally understand how that may not match up with your goals for the project.

Thanks for creating such a useful script and I look forward to hearing your thoughts!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,12 +13,24 @@
...
   "keywords": [
-    "load"
+    "javascript",
+    "load",
+    "performance"
+  ],
+  "contributors": [
+    "Scott Jehl (http://www.scottjehl.com)",
+    "Filament Group, Inc. <hello@filamentgroup.com> (https://www.filamentgroup.com)"
   ],
```
Comment (jgarber623) [inline package.json]: Somewhat annoying that `author` only accepts a string where `contributors` accepts an array…

File: scripts/build.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+'use strict';
+
+let colors = require('colors'),
+	exec = require('child_process').exec,
+	pkg = require('../package.json'),
+	contributors = pkg.contributors.map(function(contributor) {
+		return contributor.split(/(?:\s<.*>)?\s\(.*\)/)[0];
+	}).join(', '),
+	preamble = `/*! loadJS: ${pkg.description} (c) 2014-present ${contributors} (Based on http://goo.gl/REQGQ by Paul Irish). Freely distributed under the ${pkg.license} license. */`;
```
Comment (jgarber623) [inline scripts/build.js]: Using ES6's [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here since Node supports that sort of thing on the command line. :sunglasses:

**Comment by jgarber623** [comment]
Closing this one out owing to inactivity.
